ansible/roles/setup-microshift-host/tasks/main.yml

```diff
@@ -7,6 +7,13 @@
     state: present
     update_cache: true
 
+- name: install golang
+  ansible.builtin.shell: |
+    export GO_VER=1.18.7
+    curl -L -o go${GO_VER}.linux-amd64.tar.gz https://go.dev/dl/go${GO_VER}.linux-amd64.tar.gz && \
+    sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go${GO_VER}.linux-amd64.tar.gz
+    echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc && source ~/.bashrc
```

Comment:
If we're not installing 'golang' package, I would rather create a symlink from /usr/local/bin/go pointing to the new package. Let's not create a dependency on custom environment.